package.json

```
@@ -33,7 +33,7 @@
     "*.wasm"
   ],
   "dependencies": {
-    "node-addon-api": "^8.3.1",
+    "node-addon-api": "^8.5.0"",
```

This one is a bug in the migration.

```suggestion
    "node-addon-api": "^8.5.0",
```